<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inscription</class>
 <widget class="QDialog" name="Inscription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>551</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>608</width>
    <height>551</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>608</width>
    <height>551</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inscription - MyChat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	font-family : century-gothic;
	font-size : 18px;
}

QDialog
{
	background : url(:/Images/background.png);
}

QFrame
{
	background : transparent;
	border: none;
}

QToolButton
{
	background : red;
	border-radius: 60px;
}

QLabel
{
	color : black;
	border : none;
}

QPushButton
{
	border : 1px solid blue;
	border-radius : 15px;
	color : black;

}

QPushButton:hover
{
	color : white ;
	background : blue;
}

QLineEdit
{
	color : black;
	background :  white;
	border : 1px solid blue;
}

QSpinBox
{
	border :  1px solid blue;
	background : white;
	color : black;
}

QComboBox
{
	color : black;
	background : white;
	border : 1px solid blue;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>591</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century-gothic</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Formulaire d'inscription</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="inscrire">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>S'inscrire</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_nom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_prenom">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prénom</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_sexe">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>M</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexe</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinbox_age">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_mail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse mail</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retour">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Revenir à la page d'acceuil</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_utilisateur">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom d'utilisateur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de passe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmer le mot de passe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
